packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -141,10 +139,6 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         return this.dataStoreContext.IFluidHandleContext;
     }
 
-    private readonly serializer: IFluidSerializer;
-    // Back compat: deprecated in 0.53, can be removed in versions >= 0.55.
-    public get IFluidSerializer() { return this.serializer; }
```

Comment:
You might want to have deprecation in separate PR that goes first in - a bit cleaner.